app/src/main/java/com/x8bit/bitwarden/data/platform/util/ServerVersionUtils.kt

```diff
@@ -0,0 +1,46 @@
+package com.x8bit.bitwarden.data.platform.util
+
+import com.x8bit.bitwarden.data.platform.repository.ServerConfigRepository
+import kotlin.text.split
+import kotlin.text.toIntOrNull
+
+private const val VERSION_SEPARATOR = "."
+private const val SUFFIX_SEPARATOR = "-"
+
+/**
+ * Checks if the server version is greater than another provided version, returns true if it is.
+ */
+ fun isServerVersionAtLeast(serverConfig: ServerConfigRepository, version: String): Boolean {
```

Comment:
🎨 Can we change `serverConfig` argument to type `ServerConfig?` or possibly even `serverVersion: String?` instead of passing the repository?

Reply:
I ended up using `ServerConfig?` instead of another string since I kept second guessing myself on which string was which regardless of the names